src/utilities/http-client.ts

```diff
@@ -2,7 +2,7 @@ const defaultHeaders: HeadersInit = {
   'Content-Type': 'application/json',
 };
 
-const BASE_URL = 'https://localhost:5001/api';
+const BASE_URL = '/api';
```

Comment:
I think we need this to connect to the API. But we can use the server.target in the vite.config.ts

Reply:
i think having this as api is correct, it allows vite to have multiple base urls for running in different environments (which we may well want to do down the line)